docs/PRE-COMMIT.md

```diff
@@ -10,3 +10,27 @@ The pre-commit hook in this repository uses various plugins to run different kin
 1. [Pre-commit-hooks](https://github.com/pre-commit/pre-commit-hooks): Checks and fixes basic linting issues. All available hooks for pre-commit can be found [here](https://github.com/pre-commit/pre-commit-hooks#hooks-available).
 
 2. [Prettier mirror](https://github.com/pre-commit/mirrors-prettier): Checks for code formatting issues within Javascript files and fixes if errors found, like indentation, extra lines and spaces, invalid syntax, etc .
+
+### Installation
+1. Install pre-commit
+    Use `pip` to install pre-commit
+    ```sh
+    pip install pre-commit
+    ```
```

Comment:
If you see how this section gets rendered, you'll notice that "Install pre-commit" and "User `pip` to ...` are not in a separate line.
<img width="631" alt="Screenshot 2022-01-20 at 1 55 16 PM" src="https://user-images.githubusercontent.com/20169829/150300815-2fa8635c-4dd5-47ee-917c-381a85058e43.png">

Reply:
You can see how a markdown file will be rendered by click on the view file button. see reference image below. 
<img width="306" alt="Screenshot 2022-01-20 at 1 56 35 PM" src="https://user-images.githubusercontent.com/20169829/150301058-b04b021e-ea7b-47bc-bdda-8772a0344f0a.png">